src/modules/launcher/PowerLauncher/ViewModel/SettingWindowViewModel.cs

```diff
@@ -16,6 +18,28 @@ public SettingWindowViewModel()
         {
             _storage = new WoxJsonStorage<PowerToysRunSettings>();
             Settings = _storage.Load();
+
+            // Check information file for version mismatch
+            try
+            {
+                PowerToysRunSettings data = JsonSerializer.Deserialize<PowerToysRunSettings>("{}", _storage.GetInformationSerializerOptions());
```

Comment:
why deserializing {} ? why not just create instance of PowerToysRunSettings ?

Reply:
We deserialize it in order to use the newly added _informationSerializerOptions.

```
private static readonly JsonSerializerOptions _informationSerializerOptions = new JsonSerializerOptions
        {
            DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.Never,
            IncludeFields = true,
            PropertyNameCaseInsensitive = true,
            WriteIndented = true,
        };
```
Added a different DefaultIgnoreCondition to check all JSON fields.